src/iframe/storage/unpartitioned-storage/local-storage.ts

```diff
@@ -24,4 +24,22 @@ export class LocalStorage extends EnhancedStorage {
 
     return this.instance;
   }
+
+  // TODO: This is not the most elegant solution for the use case in `StorageMock` (`plasmo-storage.mock.ts`). Refactor this later:
```

Comment:
Already commented here. Ideally, all `StorageMock` / `EnhancedStorage` methods simply wait for the unpartitioned state access, but that would now require quite a bit of refactoring, so I just patched it like this.